master/pkg/model/environment_config.go

```diff
@@ -24,16 +24,19 @@ const (
 
 // Environment configures the environment of a Determined command or experiment.
 type Environment struct {
-	Image                RuntimeItem  `json:"image"`
-	EnvironmentVariables RuntimeItems `json:"environment_variables,omitempty"`
-
-	Ports          map[string]int    `json:"ports"`
+	Image          RuntimeItem       `json:"image"`
```

Comment:
appreciate the restructuring, maybe drop comments to describe the mini-sections within the struct?

Reply:
This is done in a different PR. Also, I think it's not very necessary to add comments for mini-sections since they are already self-explained.